### PR TITLE
feat(async-csv): Display SHA1 checksum on download page

### PR DESCRIPTION
**Pull request by Zylphrex**
Before
Screen Shot 2020-06-10 at 6 00 03 PM

After:
Screen Shot 2020-06-12 at 10 55 47 AM

Docs:
Screen Shot 2020-06-12 at 11 03 19 AM

getsentry/sentry-docs#1747

### CODE REVIEW

**Review by dashed — APPROVED** [review]

**Comment by joshuarli** [comment]
It might be more helpful, to directly provide a command people can run:

```bash
echo "checksum_goes_here filename" | sha1sum -c - && echo "ok"
```

**Comment by dashed** [comment]
@joshuarli I don't think that kind of copy should belong here. If the person knows what the hash means, they'll likely know how to verify it.

**Comment by joshuarli** [comment]
> @joshuarli I don't think that kind of copy should belong here. If the person knows what the hash means, they'll likely know how to verify it.

Sure, just a suggestion. I don't really know our UX patterns when it comes to these things.

**Comment by EvanPurkhiser** [comment]
I like Josh's suggestion if it was behind a toggle

**Comment by markstory** [comment]
A less technical user is going not going to know how to shasum1 a file. If you don't want to include instructions on how to check the sum, perhaps a link to the docs instead?

**Review by markstory — APPROVED** [review]
Love it ❤️

**Review by dashed — APPROVED** [review]

File: src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx [hunk 1]
```diff
@@ -158,6 +160,19 @@ class DataDownload extends AsyncView<Props, State> {
             <br />
             {this.renderDate(dateExpired)}
           </p>
+          <small>
+            <strong>SHA1:{checksum}</strong>
+          </small>
+          <p>
+            Need help verifying? Checkout our{' '}
```
Comment (k-fish) [inline src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx:167]: Noob question, does this need to be wrapped in a translation helper?

Comment (Zylphrex) [reply src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx:167]: Hm... good question, feels like it should though I'm not sure how to deal with the anchor tag within a translation helper. Maybe I should change it to a button and have 2 separate phrases? Ideas?

Comment (dashed) [reply src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx:167]: @Zylphrex there's `tct` that covers this usecase

Comment (markstory) [reply src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx:167]: It should. Good catch.

**Review by dashed — CHANGES_REQUESTED** [review]

**Review by dashed — APPROVED** [review]

**Pull request merged**